.github/workflows/build-wavefront-floware-develop.yaml

```
@@ -81,8 +85,25 @@ jobs:
           docker tag rootflo:${{ env.IMAGE_TAG }} ${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ env.IMAGE_TAG }}
           docker push ${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ env.IMAGE_TAG }}
 
+        # Configure Azure credentials and push to ACR
+      - name: Login to Azure
+        uses: azure/login@v1
```

actionlint (1.7.11)

[error] the runner of "azure/login@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)